Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang:1.12
```

Comment:
Should we use this in our travis builds, too?

Reply:
If you mean the GoLang version, I think it is always nice to be current.  If you mean the container itself, I don't see why not, but I also don't know that much about how travis works.

Reply:
I meant the container itself.

I think it's usually important that the way we do dev work mirrors exactly the way we do CI.

Could you revert the commit that adds the Docker container? I think it might be a good idea, but I'd like to do it more comprehensively, covering each of the Go versions we want to support, that sort of thing.